lib/team_api/canonicalizer.rb

```diff
@@ -9,8 +9,12 @@ class Canonicalizer
     # Canonicalizes the order and names of certain fields within site_data.
     def self.canonicalize_data(site_data)
       sort_collections site_data
```

Comment:
OK cool. Let's keep this as-is for now, but I will start a conversation about this in the up and coming 18F ruby guides so we can get consensus